wordcount_and_median.py

```diff
@@ -1,41 +1,40 @@
 import os
-from collections import Counter
 import re
+import csv
+import sys
 import bisect
-from statistics import median
-
+import statistics
+from collections import Counter
 
 
```

Comment:
Two minor style notes:

1) You don't need the ``\`` at the end of lines when in the middle of parenthesis, so I removed those.

2) The Python style guide says that you put spaces around an equals sign when doing an assignment, but NOT when defining default values on function arguments.  Go figure.